maxim/logger/groq/client.py

```
@@ -0,0 +1,231 @@
+    Args:
+        logger (Logger): The Maxim logger instance to use for tracking and
+            logging API interactions. This logger will be used to create
+            traces and generations for each API call.
+    """
+
+    global _INSTRUMENTED
+    if _INSTRUMENTED:
+        scribe().debug("[MaximSDK] Groq already instrumented")
+        return
+
+    def wrap_sync_create(create_func):
```

Ruff (0.11.9)

`wrap_sync_create` is too complex (11 > 10)

(C901)